### ISSUE TITLE
Test Set Creation

### ISSUE THREAD

**Issue by smdrnks** [Bitbol-Lab/ProtMamba-ssm#4]
Hi thank you for your work and the nice codebase. I have a question regarding the creation of the held out test set, referenced by "encoded_MSAs_test.pkl". In core.py training and validation are split on the fly with seed=0 and eval_size=193. Am I correct that the test set is sampled similarly, i.e. from the full training set (without replacement), with the same seed, and consisting of 500 clusters as described in section 2.3 in the paper?

Thank you!

**Comment by damiano-sg** [comment]
Hi, the test set is made of 500 clusters held out from the training set, but we use them in different ways, to compute the test perplexity we use all of them, to generate novel sequences instead we use only few clusters (the ones listed in the appendix). We realized that we still didn't share the full test set in this repo and we plan to do it as soon as possible!

**Comment by smdrnks** [comment]
Providing the test set would be great! Thank you for your answer.

**Comment by CyrilMa** [comment]
Hi @smdrnks, 

Thank you for your patience. We have added two new text files [https://github.com/Bitbol-Lab/ProtMamba-ssm/blob/main/data/cluster_testing_set.txt](https://github.com/Bitbol-Lab/ProtMamba-ssm/blob/main/data/cluster_testing_set.txt) and [https://github.com/Bitbol-Lab/ProtMamba-ssm/blob/main/data/cluster_training_set.txt](https://github.com/Bitbol-Lab/ProtMamba-ssm/blob/main/data/cluster_training_set.txt) here. 

Please let us know if you need anything else!
